### ISSUE TITLE
qute-pass fails to load password

### ISSUE THREAD

**Issue by choufleur** [qutebrowser/qutebrowser#8180]
**Version info**:

qutebrowser v3.1.0
Git commit: 
Backend: QtWebEngine 6.7, based on Chromium 118.0.5993.220 (from api)
Qt: 6.7.0

**Does the bug happen if you start with `--temp-basedir`?**:

no tried yet

**Description**

the title says it all: qute-pass which was working perfectly, only loads login=username, does not load password any longer, even with option --password-only

**How to reproduce**

invoke qute-pass on any site with login

**Comment by The-Compiler** [comment]
Please elaborate. What do you mean by "does not load password any longer"? When was it "working perfectly" and what changed?

**Comment by choufleur** [comment]
It was working perfectly two days ago. I am using the arch package. The password field is left blank, just the cursor blinking on it. Hard to be more specific, sorry.

**Comment by choufleur** [comment]
I found the culprit. A long time ago, when installing qutebrowser, I did 

pip install --user codext

but my installation was outdated. Updating was not easy though. I had to remove (cautiously) files in ~/.local/bin and then reinstall with 

sudo pacman -S python-pipx

then

pipx install  codext

Actually, other programs besides qute-pass  were impacted as well, for instance isync: mbsync was complaining about codext.

I am not sure if qute-pass is now packaged together with qutebrowser. It used not to be the case, at least on Fedora.

Now everything is back to normal. Thanks !
